src/openassetio-python/cmodule/CMakeLists.txt

```diff
@@ -30,8 +30,38 @@ install(
     TARGETS openassetio-python-module
     EXPORT ${PROJECT_NAME}_EXPORTED_TARGETS
     DESTINATION ${_install_subdir}
-    COMPONENT openassetio-python-module
+    COMPONENT openassetio-python-distribution
 )
+if (BUILD_SHARED_LIBS)
+    # For a shared library build, we must ensure the core C++ library
+    # dependency is installed when the Python extension module component
+    # is installed. The openassetio-python-distribution component is
+    # targeted for install when building Python wheels - see setup.py.
+    # In addition, ensure the Python bridge library is also available,
+    # to enable (the albeit rare case of) C++ plugin direct interaction
+    # with Python objects.
+    if (UNIX)
+        install(
+            TARGETS openassetio-core openassetio-python-bridge
+            COMPONENT openassetio-python-distribution
+        )
+    elseif (WIN32)
+        install(
+            TARGETS openassetio-core openassetio-python-bridge
+            COMPONENT openassetio-python-distribution
+            # On Windows we must, unfortunately, duplicate the
```

Comment:
Perhaps I'm not seeing the bigger picture here (unchanged files perhaps). But I'm not immediately seeing a duplication of DLLs.

I'm going to check build logs.

Reply:
The duplication comes because of the non-standard `DESTINATION` here.  There are also install rules for those targets independently, which will install to the default destination (`lib`/`bin`).